docs/proposals/service-catalog.md

```diff
@@ -0,0 +1,408 @@
+        I can consume in order to locate the right service to use in my
+        application
+2.  Recipes for running software systems in Kubernetes:
+    1.  As someone who created a recipe for running a software system in
+        Kubernetes, I want to share this recipe with others so that they can
+        easily stand up their own copy
+    2.  As someone who wants to run a particular software system in Kubernetes,
+        I want to be able to search for and use recipes that others may have
+        already created, so I can avoid spending time getting it to run myself
+3.  Sharing resources for a service:
+    1.  As an operator of a software system, I want to share the resources that
+        are required to use the system so that my users can easily consume
+        them in their own namespaces
+    2.  As a user of a software system running in Kubernetes, I want to consume
+        the shared resources associated with that system in my own namespace so
+        that I can use the system in my application
+4.  Sharing unique resources for a service:
+    1.  As an operator of a software system, I want to be able to generate a
+        unique resource for each user that wants to use the system so that I can
+        manage permissions granularly
+    2.  As a user of a software system, I want to get a unique set of resources
+        for system in my namespace so that I can use the system in my
+        application
+5.  Policy for viewing and using services
+    1.  As a service operator, I want to be able to describe a policy for who is
+        able to view my service, so that I can ensure that only the right users
+        have access to see that my service exists
+    2.  As a service operator, I want to be able to describe a policy for who is
+        able to use my service, so that I can ensure that users have the right
+        degree of autonomy when using my service
+6.  Consuming services - visibility
+    1.  As a service operator, I want to be able to see the users consuming my
+        service and track their usage of my service, so that I can be aware of
+        the consumers of the service and charge them according to their usage
+    2.  As an developer consuming services, I want to be able to see the
+        services are being consumed, so that I can ensure that I am consuming
+        only services that I need
+7.  Resource provisioning
+    1.  As a service operator, I want to be able to provision new resources for
+        a user when they begin to consume my service so that the user will have
+        their own resources to use for my service
+
+### Advertising services and recipes
+
+Within and without a Kubernetes cluster, there are services that users wish to
+highlight and make available to other users.  Users might also wish to publish
+recipes that allow other users to run their own services.  Some examples:
+
+1.  A user's namespace contains `etcd`, `etcd- discovery`, and `postgresql`
```

Comment:
I guess this is a subset of the list I wrote above. It would be useful to put this above the list of operator/user use cases.